docs/outgoing-webhooks.md

```diff
@@ -0,0 +1,137 @@
+# Outgoing Webhooks
+
+These are a type of integration which post data to third party URLs through a HTTP POST request.
```

Comment:
Instead of starting with "These", it's more clear if you start with "Outgoing webhooks"